eeps/eep-0058.md

```diff
@@ -1,5 +1,5 @@
     Author: Sergey Prokhorov <seriy(dot)pr(at)gmail(dot)com>
-    Status: Draft
+    Status: Final/26.0 Implemented in OTP release 26.0
     Type: Standards Track
     Created: 14-Sep-2021
     Erlang-Version: 25
```

Comment:
I'm not able to suggest the Post-History here.
```
    Post-History: 06-Sep-2023
```

Reply:
This was discussed in erlang/otp#4856 started 20-May-2021